### PR TITLE
Add raycast-today extension

### PR DESCRIPTION
**Pull request by julienR2**
<p align="center">

</p>

<h1 align="center">ToDay</h1>

<h4 align="center">
Manage multiple Notion databases right from Raycast
</h4>

## Introduction

Notion recently introduce the long awaited feature `Home`, and more specifically `My Tasks`, grouping multiple databases under one single view, giving an overview of all you current task accross your workspace.

ToDay aims a bringing the same experience right into Raycast. It gives the possibility to selected multiple databases, and offers a grouped view of all the tasks accross the selected databases.

### 🔌 Connect multiple Notion databases

After signing into your Notion, you'll be able to start connecting multiple databases.

To stay close to Notion logic, a selected database is expected to have the following properties: title, person, status, date. We detect and select automatically the right properties in your databases, although you can still adjust it.

| ➕ Add                                                                                                                                                  | 👏 Success                                                                                                                                    | 🎉 Enjoy                                                                                                                                          |
| ------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
| Add database | Success | Main View |

### 🍱 Manage you tasks

From your tasks overview, you'll be able to easily manage a task, e.g. changing its status, deleting or open it...

Manage task

### 🚀 Add new task on the fly

The search bar quickly transforms into a smart way to add a new task very fast. You can use `#` to quickly select the database you want to add the task to, or `~` to select the status.

Add task

## List of commands

- Press `↵` or `⌘ + S` to set the status.
- Press `⌘ + O` to open in Notion.
- Press `^ + X` to delete a task.

## Thanks

A huge thanks to [Hypersonic](https://github.com/raycast/extensions/tree/main/extensions/hypersonic) for their extension giving me inspiration to create this one.

## What's next?

- [ ] 👥 Support multiple accounts
- [ ] 🤹 Support multiple views
- [ ] 📱 A mobile app! (80% done)

### CODE REVIEW

File: extensions/raycast-today/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# ToDay
```
Comment (LitoMore) [inline extensions/raycast-today/README.md:1]: Worth adding some useful information to readme. You can list your features and give a introduce to your extension.

File: extensions/raycast-today/package.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "raycast-today",
+  "title": "ToDay",
+  "version": "1.0.0",
+  "description": "Manage your Notion task  from multiple databases",
```
Comment (LitoMore) [inline extensions/raycast-today/package.json:6]:
```suggestion
  "description": "Manage your Notion task from multiple databases",
```

Redundant spaces.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: extensions/raycast-today/README.md (hunk 1 above)
Comment (julienR2) [reply extensions/raycast-today/README.md]: Sorry i published a bit too fast and realised I wasn't missing some doc 😅 All fixed

File: extensions/raycast-today/package.json (hunk 2 above)
Comment (julienR2) [reply extensions/raycast-today/package.json]: Fixed 👍

**Comment by koinzhang** [comment]
Do these two extensions do the same thing?
https://www.raycast.com/reboot/hypersonic

**Comment by julienR2** [comment]
@koinzhang I definitely got strongly inspired by Hypersonic. However, Hypersonic handles only one database (you connect one, and can view and manage it).

Notion brought this new features a couple of month ago, where you could view all your databases into a same view, and my extension kind of extend Hypersonic, and bring this capabilities of grouping databases under a same view. I made a new extension for a couple of reason:
- For fun 😁 to build one from the ground up
- I'm also making a mobile app offering similar capabilities
- Hypersonic is made by a studio, so it seemed less likely that I could easily bring the changes I wanted

But regarding the last point, I should probably have ping them earlier 🙂 @prrrcl if you want to have a quick look at this extension and let me know if the changes I bring would be something you'd like to see in Hypersonic, I'd be more than happy to add those new features directly into your extension! (DM/email me if you need more info or want to discuss it further!)

(In the meantime, I'll fix the issue with the CI)

**Comment by julienR2** [comment]
Alright, I check with the team behind Hypersonic and we agreed it would be a nice adding to the existing extension.

So to avoid duplicated extensions, I'll close this one, adjust the code directly into Hypersonic one, and open a new PR with the new features!

**Pull request closed without merging**